es/lecciones/transformacion-datos-xml-xsl.md

```diff
@@ -0,0 +1,525 @@
+---
+title: Transformación de datos con XML y XSL para su reutilización
+layout: lesson
+slug: transformacion-datos-xml-xsl
+date: 2016-07-07
+translation_date: 2018-08-02
+authors:
+- M. H. Beals
+reviewers:
+- Jonathan Blaney
+- Tessa C Hauswedell
+translator:
+- Antonio Rojas Castro
+editors:
+- Adam Crymble
+translation-editor:
+- José Antonio Motilla
+translation-reviewer:
+- Pedro Carrillo
+- José Antonio Motilla
+difficulty: 1
+activity: transforming
+topics: [data-manipulation]
+review-ticket: https://github.com/programminghistorian/ph-submissions/issues/162
+original: transforming-xml-with-xsl
+redirect_from: /es/lessons/transforming-xml-with-xsl
+layout: lesson
+abstract:
```

Comment:
this needs to look like:

```
abstract: |
   "Con este...
```

Please note the `|` in line 28 as well as the tab space starting line 29

Reply:
Thanks @mdlincoln i think its done